doc/conf.py

```diff
@@ -421,6 +421,7 @@
     'theme_logo_only': False,
     'current_version': version,
     'versions': ( ("latest", "/"),
+                 ("1.14.0", "/1.14.0/"),
                  ("1.13.0", "/1.13.0/"),
                  ("1.12.0", "/1.12.0/"),
                  ("1.11.0", "/1.11.0/"),
```

Comment:
remove one release from the bottom, to not overcrowd the pull-down menu.

Reply:
Changed.